src/browser/components/buttons/index.tsx

```diff
@@ -314,28 +314,86 @@ const StyledSelectedCypherFrameButton = styled(StyledCypherFrameButton)`
   color: ${props => props.theme.secondaryButtonTextHover};
   fill: ${props => props.theme.secondaryButtonTextHover};
 `
-export const FrameButton = (props: any): JSX.Element => {
-  const { pressed, children, ...rest } = props
-  return pressed ? (
-    <StyledFrameButtonPressed {...rest}>{children}</StyledFrameButtonPressed>
-  ) : (
-    <StyledFrameButton {...rest}>{children}</StyledFrameButton>
-  )
+
+type FrameButtonProps = {
+  onClick: () => void
+  children: React.ReactNode
+  title: string
 }
 
-const StyledFrameButton = styled.li`
-  color: ${props => props.theme.secondaryButtonText};
+export const FrameButton = ({
+  onClick,
+  children,
+  title
+}: FrameButtonProps): JSX.Element => (
+  <StyledFrameButton onClick={onClick} title={title}>
+    {children}
+  </StyledFrameButton>
+)
+
+type FrameControlButtonProps = {
+  pressed?: boolean
+  onClick: () => void
+  children: React.ReactNode
+  title: string
+}
+
+export const FrameControlButton = ({
+  title,
+  pressed,
+  onClick,
+  children
+}: FrameControlButtonProps): JSX.Element => (
+  <StyledFrameControlButton title={title} pressed={pressed} onClick={onClick}>
+    {children}
+  </StyledFrameControlButton>
+)
+
+const StyledFrameControlButton = styled.li<{
+  pressed?: boolean
+  size?: number
+}>`
+  border-radius: 2px;
+  color: ${props => props.theme.frameControlButtonTextColor};
+  background-color: transparent;
+  border-left: transparent;
+  height: 20px;
+  width: 20px;
+  cursor: pointer;
+  overflow: hidden;
+  text-align: center;
+  line-height: 20px;
+  display: inline-block;
+  margin-left: 12px;
+  &:hover {
+    background-color: ${props => props.theme.frameButtonHoverBackground};
+    color: ${props => props.theme.secondaryButtonTextHover};
+    fill: ${props => props.theme.secondaryButtonTextHover};
+    text-decoration: none;
+  }
+  ${props =>
```

Comment:
I have mixed feelings about the readability of this, and whether it was clearer with the original distinction with 2 separate styled components. but so long as this syntax works properly I guess it's fine.

Reply:
I think it's a lot more readable to write:
  `background-color: ${props => props.pressed ? props.theme.frameButtonHoverBackground} : 'transparent'}`
 
 Than override in the way I've done here.